models/Bounty.js

```diff
@@ -0,0 +1,59 @@
+import mongoose from 'mongoose'
+
+/* BountyBoardSchema will correspond to a collection in your MongoDB database. */
+const BountyBoardSchema = new mongoose.Schema({
```

Comment:
Going to paste the directly the files should be moved under packages/react-app/src